tests/test_config.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from cleanlab_tlm.errors import TlmBadRequestError
+from cleanlab_tlm.tlm import TLM
+from cleanlab_tlm.utils.config import (
+    get_default_context_limit,
+    get_default_model,
+)
+from tests.constants import (
+    CHARACTERS_PER_TOKEN,
+)
+
+tlm_with_default_setting = TLM()
+
+
+def assert_get_default_model_matches_tlm_default_setting(tlm: TLM) -> None:
+    assert tlm.get_model_name() == get_default_model()
+
+
+def test_prompt_too_long_exception_single_prompt(tlm: TLM) -> None:
+    """Tests that bad request error is raised when prompt is too long when calling tlm.prompt with a single prompt."""
+    with pytest.raises(TlmBadRequestError) as exc_info:
+        tlm.prompt(
+            "a" * (get_default_context_limit() + 1) * CHARACTERS_PER_TOKEN,
```

Comment:
how you decide on this string for the test?  It will produce a string like:
'aaaaaaaaa' which may be tokenized in varying ways over the next years as tokenizers evolve and thus this test will get outdated.

Can you choose a string that will be more stably tokenized?  I vote for something like this:

"no no no no no " ... whose tokenization is unlikely to change

Reply:
I didn't have to decide on this test as I re-used an existing [one](https://github.com/cleanlab/cleanlab-tlm/blob/main/tests/test_validation.py#L207).
I supposed it was approved and confirmed with Hui to re-use it.

I do prefer to change it with a word (no or hello) which would translate to a fix number of tokens (obviously given the tokenizer we use to count, and the tokenizers used by model provider).

I'll make this change to this test. Do you want me to update the existing test (linked above) as well?

Reply:
Multiple tests use `"a" * max_context * char_per_token` logic. Updating them all with `"no "` broke some tests.
Learnt that some tests depend on character based tokenization check and some uses tiktoken to calculate true number of tokens.

Hence, used "hello " instead of "no " as it suffices for both the above kind of tests.